ep/lh/ep-lh-0002.md

```diff
@@ -38,22 +59,104 @@ This is the minimal set of properties that allows to identify the engine, unders
 
 The corresponding schema should be located at [`./schema/meta.json`](#references).
 
+
+We support the following:
+<details>
```

Comment:
I suggest isolating those list in a separate EP. Please also try to avoid any tags, just Markdown syntax.